websrc/gadget_menu/src/gadget_menu_main.tsx

```diff
@@ -64,9 +66,25 @@ function GadgetInfoPanel( props: GadgetInfoPanelProps )
 								onSelect={ props.deleteCallback }/>
 			</AvTransform>
 		}
+		{ props.manifest.aardvark.startAutomatically && props.highlight == GadgetSeedHighlight.Menu &&
+			autoLaunchSubMenu(props.autoLaunchSet, props.autoLaunchCallBack)
+		}
 	</>;
 }
 
+function autoLaunchSubMenu( autoLaunchSet: boolean, autoLaunchCallBack: () => void)
+{
+	const rotationAboutModelSpaceX = autoLaunchSet ? 0 : -90;
```

Comment:
This is kind of unfortunate. I assume it's because the two icon models are rotated 90 degrees from each other.

Maybe the "icon" constant above should actually have rotation and scale as part of it, in addition to just the model name. Then a "render icon" function could turn that into a transform and a model. (Or transform and menu item like below.)

Or maybe we should go normalize the rotation and scale of all the icon models.